test cases/common/139 simd/meson.build

```diff
@@ -0,0 +1,31 @@
+project('simd', 'c')
+
+simd = import('simd')
+
+cc = meson.get_compiler('c')
+
+cdata = configuration_data()
+
+simdlibs = simd.check('mysimds',
```

Comment:
Instead of returning a static library, wouldn't it be simpler to return the source list corresponding to the 'best' instruction set that was found as `files()`? That way you don't have to duplicate kwargs from `build_target()` in here and people can add the sources wherever they like.